akka-javasdk/src/main/scala/akka/javasdk/impl/workflow/ReflectiveWorkflowRouter.scala

```diff
@@ -115,18 +114,9 @@ class ReflectiveWorkflowRouter[S, W <: Workflow[S]](
           }
           result.asInstanceOf[Workflow.Effect[_]]
         } else {
-
-          // FIXME can be proto from http-grpc-handling of the static es endpoints
```

Comment:
don't we have the same thing in other routers?

Reply:
Already removed from Consumer and TimedActions